game-core/src/main/java/games/strategy/engine/framework/map/file/system/loader/AvailableGamesFileSystemReader.java

```diff
@@ -44,8 +44,9 @@ public static synchronized AvailableGamesList parseMapFiles() {
     return availableGamesListCache;
   }
 
-  public static void populateAvailableMapFilesCache() {
-    parseMapFiles();
+  public static void refreshMapFileCache() {
+    availableGamesListCache = null;
```

Comment:
Codacy found an issue: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6221285)